x/decentralized-vault/genesis_test.go

```diff
@@ -0,0 +1,29 @@
+package decentralizedvault_test
+
+// import (
+// 	"testing"
+
+// 	keepertest "github.com/UnUniFi/chain/testutil/keeper"
+// 	"github.com/UnUniFi/chain/testutil/nullify"
+// 	"github.com/UnUniFi/chain/x/decentralized-vault"
+// 	"github.com/UnUniFi/chain/x/decentralized-vault/types"
+// 	"github.com/stretchr/testify/require"
+// )
+
+// func TestGenesis(t *testing.T) {
```

Comment:
I think you can add either genesis tests or remove this file

Reply:
I created issue.
I will implement when I have time.

https://github.com/UnUniFi/chain/issues/271